v7/create_error_pages/create_error_pages.py

```diff
@@ -0,0 +1,57 @@
+from nikola.plugin_categories import Task
```

Comment:
Add a copyright notice and license of your choice. (Steal the header from Nikola and change the names if you don’t have a favorite license.)